addons/sourcemod/scripting/vsh/bosses/boss_announcer.sp

```diff
@@ -428,9 +518,34 @@ public Action Timer_AnnouncerChangeTeam(Handle hTimer, int iClient)
 		}
 	}
 	
+	//Allow sentries to target this fella from now on
+	SetEntityFlags(iClient, (GetEntityFlags(iClient) & ~FL_NOTARGET));
+	
 	//Dont give health overheal
 	SetEntProp(iClient, Prop_Send, "m_iHealth", SDK_GetMaxHealth(iClient));
 	
 	//TF2_AddCondition(iClient, TFCond_Buffed, TFCondDuration_Infinite);
 	TF2_AddCondition(iClient, TFCond_DefenseBuffed, TFCondDuration_Infinite);
+}
+
+public void Announcer_ShowAnnotation(int iTarget, char[] sMessage, float flDuration)
+{
+	int[] iClients = new int[MaxClients];
+	int iCount = 0;
+	
+	for (int iClient = 1; iClient <= MaxClients; iClient++)
+	{
+		if (IsClientInGame(iClient))
+		{
+			SaxtonHaleBase boss = SaxtonHaleBase(iClient);
+			
+			if (iClient != iTarget && (TF2_GetClientTeam(iClient) != TFTeam_Attack || boss.CallFunction("IsBossType", "CAnnouncerMinion")))
```

Comment:
`IsBossType` may need `boss.bValid` check, just to be sure.
`iClient != iTarget` can move next to `IsClientInGame(iClient) check`